src/Components/Postulant/Interviews/index.js

```diff
@@ -1,12 +1,120 @@
-import React from 'react';
+import { useEffect, useState } from 'react';
 import styles from './interviews.module.css';
+import Modal from 'Components/Shared/Modal';
+import Table from 'Components/Shared/Table';
+import { useDispatch, useSelector } from 'react-redux';
+import { getInterviews, updateInterview } from 'redux/interviews/thunks';
+import { cleanError } from 'redux/interviews/actions';
+
+function Interviews() {
+  const [showModal, setShowModal] = useState(false);
+  const [selectedIdInterview, setIdInterview] = useState('');
+  const [disableArrayValue, setDisableArrayValue] = useState([]);
+  const [interviewsOfOnePostulant, setInterviewsOfOnePostulant] = useState([]);
+
+  const dispatch = useDispatch();
+
+  const interviews = useSelector((store) => store.interviews.list);
+  const error = useSelector((store) => store.interviews.error);
+  const isLoading = useSelector((store) => store.interviews.isFetching);
+
+  let disableArray = [];
+
+  useEffect(() => {
+    if (!interviews.length) {
+      dispatch(getInterviews());
+    }
+
+    interviewsOfOnePostulant.map((interview) => {
+      if (interview.status === 'assigned' || interview.status === 'successful') {
+        disableArray.push('false');
+      } else {
+        disableArray.push('true');
+      }
+    });
+
+    setDisableArrayValue(disableArray);
+  }, []);
+
+  useEffect(() => {
+    setInterviewsOfOnePostulant(
+      interviews.filter(
+        (interview) => interview.postulant._id === process.env.REACT_APP_POSTULANT_ID
+      )
+    );
+  }, [interviews]);
+
+  const editInterview = (selectedIdInterview) => {
+    return interviewsOfOnePostulant.find((interview) => interview._id === selectedIdInterview);
+  };
 
-const Interviews = () => {
   return (
     <section className={styles.container}>
-      <h2>Interviews</h2>
+      <h2 className={styles.title}>My Interviews</h2>
+      {isLoading ? (
+        <p className={styles.loading}>On Loading ...</p>
+      ) : (
+        <Table
+          columns={[
+            { name: 'Client', value: 'client.name' },
+            { name: 'Status', value: 'status' },
+            { name: 'Date', value: 'date' }
+          ]}
+          data={interviewsOfOnePostulant}
+          actions={[
+            {
+              text: 'Cancel',
+              disabled: (item) => item.status !== 'assigned' && item.status !== 'confirmed',
+              callback: (e, item) => {
+                e.stopPropagation();
+                if (item.status === 'assigned' || item.status === 'confirmed') {
+                  setIdInterview(item._id);
+                  setShowModal(true);
+                }
+              }
+            }
+          ]}
+        />
```

Comment:
Missing props onRowClick. I suggested to pass an empty function or to Adapt a new table to get rid of the onClick function. In case of adding a new table component, get rid of the css module pointer on row:hover.